Dockerfile

```diff
@@ -19,166 +20,291 @@
+
+#------------------------------------------------------------------------
+# REPO / ANDROID SDK
+FROM build AS androidbuild
+
+ARG ANDROID_SDK_VERSION=6858069
+ENV ANDROID_HOME=/opt/android-sdk
+
+RUN curl -ksS https://storage.googleapis.com/git-repo-downloads/repo > /usr/bin/repo \
+    && chmod a+x /usr/bin/repo
+
+RUN curl -Os https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip \
+    && unzip -q commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip -d $ANDROID_HOME \
+    && rm commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip \
+    && PROXY_HOST_AND_PORT=${https_proxy#*://} \
+    && if [ -n "$PROXY_HOST_AND_PORT" ]; then \
+       # While sdkmanager uses HTTPS by default, the proxy type is still called "http".
+       SDK_MANAGER_PROXY_OPTIONS="--proxy=http --proxy_host=${PROXY_HOST_AND_PORT%:*} --proxy_port=${PROXY_HOST_AND_PORT##*:}"; \
+    fi \
+    && yes | $ANDROID_HOME/cmdline-tools/bin/sdkmanager $SDK_MANAGER_PROXY_OPTIONS \
+       --sdk_root=$ANDROID_HOME "platform-tools" "cmdline-tools;latest"
+COPY docker/android.sh /etc/ort/bash_modules
+
+#------------------------------------------------------------------------
+# SCANCODE - Build scancode as a separate component
+FROM pythonbuild AS scancodebuild
+
+ARG SCANCODE_VERSION="3.2.1rc2"
+ENV SCANCODE_URL "https://github.com/nexB/scancode-toolkit/archive/refs/tags/"
+ENV PATH=/opt/python/shims:$PATH
+
+RUN mkdir -p /opt/scancode \
+    && curl -ksSL ${SCANCODE_URL}/v${SCANCODE_VERSION}.tar.gz | tar -C /opt/scancode -xz --strip-components=1 \
+    && cd /opt/scancode \
+    && PYTHON_EXE=python3 ./configure \
+    # cleanup unneeded installed binaries
+    && rm -rf /opt/scancode/thirdparty
+
+#------------------------------------------------------------------------
+# Main container
+FROM eclipse-temurin:11-jre
+
+# ORT
+COPY --from=ortbuild /opt/ort /opt/ort
+COPY docker/bash_bootstrap.sh /etc/ort/bash_bootstrap.sh
+COPY docker/ort-wrapper.sh /usr/bin/ort
+RUN chmod 755 /usr/bin/ort
+
+# Python
+COPY --from=pythonbuild /opt/python /opt/python
+COPY --from=pythonbuild /etc/ort/bash_modules/python.sh /etc/ort/bash_modules/
+
+# Ruby
+COPY --from=rubybuild /opt/rbenv /opt/rbenv
+COPY --from=rubybuild /etc/ort/bash_modules/ruby.sh /etc/ort/bash_modules/
+
+# NodeJS
+COPY --from=nodebuild /opt/nodejs /opt/nodejs
+COPY --from=nodebuild /etc/ort/bash_modules/nodejs.sh /etc/ort/bash_modules/
+
+# Golang
+COPY --from=gobuild /opt/go /opt/go/
+COPY --from=gobuild  /etc/ort/bash_modules/go.sh /etc/ort/bash_modules/
+
+# Haskell
+COPY --from=haskellbuild /usr/bin/stack /usr/bin
+
+# Repo and Android
+COPY --from=androidbuild /usr/bin/repo /usr/bin/
+COPY --from=androidbuild /opt/android-sdk /opt/android-sdk
+COPY --from=androidbuild /etc/ort/bash_modules/android.sh /etc/ort/bash_modules/
+
+# Scancode
```

Comment:
Change `Scancode` to `ScanCode` which is official spelling per https://github.com/nexB/scancode-toolkit